toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -532,7 +532,6 @@ func initializeRpmDatabase(installRoot string, diffDiskBuild bool) (err error) {
 			return err
 		}
 	}
-	err = initializeTdnfConfiguration(installRoot)
```

Comment:
This looks like a significant change. I'd definitely verify the VHD, ISO, and container images, if they still build correctly and TDNF runs without issues.

Reply:
I have currently tested ISO and vhdx images locally with a change to how the `mariner-release` package is included in the image. Rather than installing `mariner-release` ahead of all packages using rpm, the package will instead be added to the list of packages to build giving the same functionality without the reliance on `tdnf` not detecting packages as already installed if done so via rpm.